packages/sedlex/sedlex.3.2/opam

```diff
@@ -0,0 +1,48 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "An OCaml lexer generator for Unicode"
+description: """
+sedlex is a lexer generator for OCaml. It is similar to ocamllex, but supports
+Unicode. Unlike ocamllex, sedlex allows lexer specifications within regular
+OCaml source files. Lexing specific constructs are provided via a ppx syntax
+extension."""
+maintainer: ["Alain Frisch <alain.frisch@lexifi.com>"]
```

Comment:
Would it be fair to say that you are now the maintainer? Looking at the graphs from github it certainly looks that way…

Reply:
The package and module are maintained by the `ocaml-community` group.

Reply:
Ok, I guess we can either leave the field as is (it's not very accurate but also not completely arbitrary) or update it to "ocaml-community" (is there an email address associated to that entity?). I don't mind either.

Reply:
Good question. I don't think that there's an email. I would leave it as is for now, it's something I should at least discuss with the community before making a decision.

Reply:
:+1: